src/transformer/project-bundle.ts

```diff
@@ -110,3 +111,15 @@ export const findManuscriptById = (
     `There is no manuscript found for the following _id (${manuscriptID})`
   )
 }
+
+const isTitle = hasObjectType<Title>(ObjectTypes.Title)
+
+export const findTitle = (modelMap: Map<string, Model>): Title => {
+  for (const model of modelMap.values()) {
+    if (isTitle(model)) {
+      return model
+    }
+  }
+
+  throw new Error('No Title found')
```

Comment:
I think in this case we should have a default title Model. CC @mbartenev-atypon

Reply:
done